src/org/opendatakit/briefcase/model/BriefcaseFormDefinition.java

```diff
@@ -365,6 +366,15 @@ public boolean isFileEncryptedForm() {
   }
 
   public TreeElement getSubmissionElement() {
+    TreeElement treeElement = formDefn.getSubmissionElement();
```

Comment:
Just looking through the code (which I admittedly don't fully understand), it looks like BriefcaseFormDefinition.getSubmissionKey(String uri) may also need this lazy-load treatment...its value depends on formDefn.getSubmissionKey(uri) which depends on submissionElement.getName(); submissionElement is transient.